packages/mock-addon/src/components/MockItem/index.js

```diff
@@ -26,28 +21,34 @@ const ObjectContent = styled.div`
 `;
 
 const Method = styled.div`
-    padding: 0.75rem;
     font-weight: 700;
-    border-right: 1px solid #ddd;
+    border-right: 1px solid ${({ theme }) => theme.appBorderColor};
 `;
 
 const Url = styled.div`
-    padding: 0.75rem;
     flex: 1;
+    overflow-x: auto;
 `;
 
 const UrlMethodContainer = styled.div`
     display: flex;
     align-items: center;
-    border: 1px solid #ddd;
-    background: #eee;
-    border-radius: 5px;
+    border: ${({ theme }) => theme.input.border};
+    background: ${({ theme }) => theme.input.background};
+    border-radius: ${({ theme }) => theme.input.borderRadius};
```

Comment:
Color-theme-style changes are necessary since auto dark mode is also now integrated by storybook v7